### PR TITLE
Bug 1268939   Login records should be removed if user signs out

### PR DESCRIPTION
**Pull request by jacobwhite**
Asks the user if they want to remove logins when signing out of Firefox.

### CODE REVIEW

File: Client/Frontend/Settings/AppSettingsOptions.swift [hunk 1]
```diff
@@ -70,6 +70,20 @@ class DisconnectSetting: WithAccountSetting {
                 self.settings.profile.removeAccount()
                 self.settings.settings = self.settings.generateSettings()
                 self.settings.SELfirefoxAccountDidChange()
+                let loginsController = UIAlertController(
+                    title: NSLocalizedString("Remove Logins?", comment: "Title of the 'Remove firefox logins' alert"),
```
Comment (rnewman) [inline Client/Frontend/Settings/AppSettingsOptions.swift]: "firefox" throughout should be a brand reference. I think we're also using string identifiers now; @thebnich?

Comment (thebnich) [inline Client/Frontend/Settings/AppSettingsOptions.swift]: Yes, please! For new strings, please use `NSLocalizedString` with the `value` parameter, where the first unnamed parameter is a unique, namespaced ID, and the `value` parameter is the string itself. [Here's one example.](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Strings.swift#L21)

**Comment by jacobwhite** [comment]
Should I fix the strings in the other alert while I'm at it?

**Comment by thebnich** [comment]
No, since the new IDs would mean localizers would have to go through and update the strings for each language. At some point in the future, we hope to go through and batch update all of these places that use the obsolete string format.

**Comment by farhanpatel** [comment]
Thanks for this PR! We just finished a major migration to swift 3.0 making merging in most patches pretty difficult. If you feel like this patch is still important please feel free to update your PR to swift 3 and reopen this PR 😄

**Pull request closed without merging**